data-es7.js

```diff
@@ -78,6 +78,16 @@ exports.browsers = {
   firefox41: {
     full: 'Firefox',
     short: 'FF 41',
+    obsolete: true,
```

Comment:
Perhaps we need to combine es5, es6, es7 data into 1 file, to avoid having to update browsers' status like this (across multiple tables).